app/controllers/admin/project_groups_controller.rb

```diff
@@ -23,14 +34,40 @@ def create
     end
   end
 
+  def update
+    if @entry.update(project_group_params)
+      render nothing: true
+    else
+      render_modal_partial(status: 422)
+    end
+  end
+
+  def edit
+    render_modal_partial
+  end
+
+  def destroy
+    if @entry.destroy
+      render nothing: true
+    else
+      render_modal_partial(status: 422)
```

Comment:
This doesn't make sense to me. Why would we render the modal partial? I don't think we are ever expecting destroy to fail, right? So I would actually raise an error if it does perhaps?